File: test/fixture-java/Service.java
// Cross-file callee. Service.handle (a method) calls util (a method in the same
// class/file) — the leaf the trace should reach from App.run().
package demo;

class Service {
    int util(int x) {
        return x * 2;
    }

    int handle(int n) {
        return util(n);
    }
}

File: test/fixture-kotlin/Service.kt
// Cross-file callee. Service.handle (a method) calls util (a top-level function
// in the same file) — the leaf the trace should reach from run().
package demo

fun util(x: Int): Int = x * 2

class Service {
    fun handle(n: Int): Int {
        return util(n)
    }
}
